plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF

```diff
@@ -7,6 +7,10 @@ Bundle-Vendor: Google, Inc.
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ActivationPolicy: lazy
-Export-Package: com.google.cloud.tools.eclipse.util.io,
+Export-Package: com.google.cloud.tools.eclipse.util,
+ com.google.cloud.tools.eclipse.util.io,
  com.google.cloud.tools.eclipse.util.status
 Require-Bundle: org.eclipse.core.runtime
+Import-Package: com.google.cloud.tools.appengine.cloudsdk.process,
```

Comment:
why do we need to import this? I would expect the util package not to have any dependencies on our other non-util packages.